Configuration/Settings.yaml

```diff
@@ -1,17 +1,21 @@
 Neos:
   Cqrs:
     EventStore:
-      storage:
-        options:
-          eventTableName: 'neos_cqrs_events'
-
-          backendOptions:
-            driver: 'pdo_mysql'
-            host: '127.0.0.1'
-            dbname: NULL
-            user: NULL
-            password: NULL
-            charset: 'utf8'
+      stores:
+        'default':
+          boundedContexts:
+            'Foo.Bar': true
+            '*': true
+          storage: '...DoctrineLogger'
+          storageOptions:
+            eventTableName: 'neos_cqrs_events'
+            backendOptions:
+              driver: 'pdo_mysql'
+              host: '127.0.0.1'
+              dbname: NULL
+              user: NULL
+              password: NULL
+              charset: 'utf8'
```

Comment:
hm, I'd say we solve this in the Doctrine Connector; if no DB name ... is specified, we use the Flow connection out of the box. What do you think?